exchanges/bybit/bybit_wrapper.go

```diff
@@ -0,0 +1,1826 @@
+			side = orderCancellation.Side.Title()
+		}
+
+		successful, err := by.BatchCancelOrder(ctx, orderCancellation.Pair.String(), side, orderType)
+		if !successful {
+			return cancelAllOrdersResponse, fmt.Errorf("failed to cancelAllOrder")
+		}
+		if err != nil {
+			status = err.Error()
+		}
+		for i := range activeOrder {
+			cancelAllOrdersResponse.Status[strconv.FormatInt(activeOrder[i].OrderID, 10)] = status
+		}
+
+	case asset.CoinMarginedFutures:
+		resp, err := by.CancelAllActiveCoinFuturesOrders(ctx, orderCancellation.Pair)
+		if err != nil {
+			status = err.Error()
+		}
+		for i := range resp {
+			cancelAllOrdersResponse.Status[resp[i].OrderID] = status
+		}
+	case asset.USDTMarginedFutures:
+		resp, err := by.CancelAllActiveUSDTFuturesOrders(ctx, orderCancellation.Pair)
+		if err != nil {
+			status = err.Error()
+		}
+		for i := range resp {
+			cancelAllOrdersResponse.Status[resp[i]] = status
+		}
+	case asset.Futures:
+		resp, err := by.CancelAllActiveFuturesOrders(ctx, orderCancellation.Pair)
+		if err != nil {
+			status = err.Error()
+		}
+		for i := range resp {
+			cancelAllOrdersResponse.Status[resp[i].CancelOrderID] = status
+		}
+	case asset.USDCMarginedFutures:
+		activeOrder, err := by.GetActiveUSDCOrder(ctx, orderCancellation.Pair, "PERPETUAL", "", "", "", "", "", 0)
+		if err != nil {
+			return cancelAllOrdersResponse, err
+		}
+
+		err = by.CancelAllActiveUSDCOrder(ctx, orderCancellation.Pair, "Order")
+		if err != nil {
+			status = err.Error()
+		}
+		for i := range activeOrder {
+			cancelAllOrdersResponse.Status[activeOrder[i].ID] = status
+		}
+	default:
+		return cancelAllOrdersResponse, fmt.Errorf("%s %w", orderCancellation.AssetType, asset.ErrNotSupported)
+	}
+	return cancelAllOrdersResponse, nil
+}
+
+// GetOrderInfo returns order information based on order ID
+func (by *Bybit) GetOrderInfo(ctx context.Context, orderID string, pair currency.Pair, assetType asset.Item) (order.Detail, error) {
+	switch assetType {
+	case asset.Spot:
+		resp, err := by.QueryOrder(ctx, orderID, "")
+		if err != nil {
+			return order.Detail{}, err
+		}
+
+		return order.Detail{
+			Amount:         resp.Quantity,
+			Exchange:       by.Name,
+			OrderID:        strconv.FormatInt(resp.OrderID, 10),
+			ClientOrderID:  resp.OrderLinkID,
+			Side:           getSide(resp.Side),
+			Type:           getTradeType(resp.TradeType),
+			Pair:           pair,
+			Cost:           resp.CummulativeQuoteQty,
+			AssetType:      assetType,
+			Status:         getOrderStatus(resp.Status),
+			Price:          resp.Price,
+			ExecutedAmount: resp.ExecutedQty,
+			Date:           resp.Time.Time(),
+			LastUpdated:    resp.UpdateTime.Time(),
+		}, nil
+
+	case asset.CoinMarginedFutures:
+		resp, err := by.GetActiveRealtimeCoinOrders(ctx, pair, orderID, "")
+		if err != nil {
+			return order.Detail{}, err
+		}
+
+		if len(resp) != 1 {
+			return order.Detail{}, fmt.Errorf("invalid order's count found")
```

Comment:
```suggestion
			return order.Detail{}, fmt.Errorf("expected 1 order, received %v orders", len(resp))
```
Please do this for the others too

Alternatively, you could define an `errExpectedOneOrder = errors.New("expected one order")`
and then this becomes
```suggestion
			return order.Detail{}, fmt.Errorf("%w, received %v orders", errExpectedOneOrder,  len(resp))
```